flybywiresim/a32nx
# feat: Added instrument hints to PRELIMINARY COCKPIT PREP. checklist page

## Summary of Changes
Added instrument hints (eye icon) for the PRELIMINARY COCKPIT PREP. checklist page.
Further, the camera view is not changed for three already implemented instrument hints.

Instrument hints added
- Battery switches
- External power
- APU Fire Test
- APU start
- Cockpit Lights
- APU bleed
- X-Bleed

Camera view fixed
- Wipers
- APU master switch
- Probe / Window Heat


## Screenshots (if necessary)


![missing_instrument_hint](https://user-images.githubusercontent.com/7311836/109692041-58c4e700-7b88-11eb-8d5d-8fa0dc4044f1.png)
![added_instrument_hint](https://user-images.githubusercontent.com/7311836/109692054-5b274100-7b88-11eb-9696-a1ac6297620e.png)



## References





## Additional context



Discord username (if different from GitHub):

## Testing instructions
- Check if the "eye icon" appears for
  - Battery switches
  - External power
  - APU Fire Test
  - APU start
  - Cockpit Lights
  - APU bleed
  - X-Bleed
- Check if the camera view is changed to the instrument location when pushing the "eye icon"
  - Wipers
  - APU master switch
  - Probe / Window Heat


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
